src/index.html

```diff
@@ -1,14 +1,80 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet">
+  </head>
+  <body>
+    <header class="header">
+
+      <div class="logo">
+        <a href="#mac">
+          <img src="./images/logo.png"
+            alt="logo"
+          >
+        </a>
+      </div>
```

Comment:
```suggestion
      <a href="#mac"  class="logo">
        <img src="./images/logo.png"
          alt="logo"
        >
      </a>
```

No need for this `div` wrapper. You can just apply class 'logo' to tour 'a' tag.
Btw, what should the name `mac` mean? 🤔